pkg/tracing/tracing.go

```diff
@@ -99,3 +98,15 @@ func splitTagSettings(input string) map[string]string {
 
 	return res
 }
+
+type jaegerLogWrapper struct {
```

Comment:
what's the purpose of this?

Reply:
The logger we use in Grafana does not match the interface in jaeger. So I wrote a small facade/wrapper to enabled jaeger to write to Grafanas log.